nucypher/utilities/prometheus/collector.py

```diff
@@ -121,6 +124,10 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
+        if self.ursula._availability_check:
```

Comment:
I don't believe that we can only use this check here. There are two determinants of whether the availability check is run:
1. The ursula configuration file (self.ursula._availability_check)
2. The `ursula run` CLI option (--availability-check / --disable-availability-check)

Therefore `self.ursula._availability_check` could be true, but the CLI option `--disable-availability-check` is specified, in which case the availability tracker is not run. These are the two values checked here - https://github.com/nucypher/nucypher/blob/main/nucypher/characters/lawful.py#L1214

I think you can instead use

```suggestion
        if self.ursula._availability_tracker.running:
```

@KPrasch interested in your thoughts here.

Reply:
`self.ursula._availability_tracker.running:` does look like it would be the better source of truth here, at least looking at [this block](https://github.com/nucypher/nucypher/blob/76eea579a3b94e1ea2a8019601e0b454f81e5e2c/nucypher/cli/painting/nodes.py#L84-L88):

```
    if ursula._availability_tracker:
        if ursula._availability_tracker.running:
            score = 'Availability Score .. {} ({} responders)'.format(ursula._availability_tracker.score, len(ursula._availability_tracker.responders))
        else:
            score = 'Availability Score .. Disabled'
```

We may also want to use that same nested conditional:
```
if ursula._availability_tracker:
    if ursula._availability_tracker.running:
```
to prevent any TypeErrors

Reply:
Tried this out with https://github.com/nucypher/nucypher/pull/2264/commits/3fd1f8b267c4043e4c8c7d9ce015dffb1aca43ae